src/examples/src/customers/createcard/create.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * Mercado Pago Customer card create.
+ *
+ * @see {@link https://www.mercadopago.com.br/developers/pt/reference/cards/_customers_customer_id_cards/post Documentation}.
+*/
+import { Customer } from '@src/clients/customers';
```

Comment:
nos exemplos, sempre fazer a importação do client a partir do `index`

Reply:
feito

Reply:
aqui nao mudou, mas eu to dizendo que tem que ficar desse jeito: 

`import MercadoPago, { Customer } from '@src/index';`

da uma olhada no outro exemplo que vc fez, mais ou menos daquela maneira